### PR TITLE
Tsm

### PR DESCRIPTION
**Pull request by fishkerez**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (FrimIdan) [inline .github/ISSUE_TEMPLATE/bug_report.md]: don't think that we need this file, it is APIClarity deployment related.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: CI
+on:
+  pull_request:
+
+env:
+  GO_VERSION: 1.17
+
+jobs:
+
+  build:
```
Comment (FrimIdan) [inline .github/workflows/ci.yml:10]: `build` --> `verification`

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    name: Build
```
Comment (FrimIdan) [inline .github/workflows/ci.yml:11]: `Build` --> `Verification`

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+.idea/
```
Comment (FrimIdan) [inline .gitignore:1]: please also add `bin/` and delete first the `bin/` content that was pushed.

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+	@(echo "Generating API code ..." )
+	@(cd api; ./generate.sh)
+
+.PHONY: test
+test: ## Run Unit Tests
+	@(cd manager && go test ./pkg/...)
+
+.PHONY: clean
+clean: ## Clean all build artifacts
+	@(rm -rf manager/bin/* ; echo "Build artifacts cleanup done" )
```
Comment (FrimIdan) [inline Makefile]: remove

File: bin/golangci-lint [hunk 6]
```diff
@@ -0,0 +1 @@
+golangci-lint-1.42.0
```
Comment (FrimIdan) [inline bin/golangci-lint]: please remove and also remove `bin/golangci-lint-1.42.0`

**Comment by akpsgit** [comment]
Can you please refer to the changed code(what was not copy-paste)?

**Comment by fishkerez** [comment]
@akpsgit 
What was changed:
cmd folder removed
Added github folder for workflows
In grpc proto: remove ComponentID enum, instead using string
in initHostToTrace: Dont return error if secret not found (dont force to use secret)

**Pull request merged**